app.js

```diff
@@ -11,17 +11,18 @@ var http = require('http');
 var session = require('express-session');
 
 var log_level = 'info';
-if (process.env.NODE_ENV == 'dev') {
+if (process.env.NODE_ENV == 'dev' || process.env.DEBUG) {
     log_level = 'debug';
 }
 
 var winston = require('winston');
 var jwt = require('jsonwebtoken');
 
 const myconsole = new (winston.transports.Console)({
-      timestamp: true
+  label: 'gomngr',
+  level: log_level
 });
-winston.loggers.add('gomngr', {
+const wlogger = winston.loggers.add('gomngr', {
```

Comment:
add function returns a logger itself?

could you check wlogger outputs were as expected?

Reply:
It look like it return this:
https://github.com/winstonjs/winston/blob/96591977288f7f1cd1bf38b7e8e629abb3e5f725/lib/winston/logger.js#L480
Do you want I add a check of the type in the code ?